app/experimenter/settings.py

```diff
@@ -278,6 +279,12 @@
 CELERY_BROKER_URL = "redis://{host}:{port}/{db}".format(
     host=REDIS_HOST, port=REDIS_PORT, db=REDIS_DB
 )
+CELERY_BEAT_SCHEDULE = {
+    "debug_task": {
+        "task": "experimenter.experiments.tasks.update_experiment_status",
+        "schedule": 10,
```

Comment:
Was not sure what would a reasonable time interval would be

Reply:
Yeah I think 5 minutes is probably fine?